docs/ai/quickstarts/build-mcp-server.md

```diff
@@ -85,30 +78,219 @@ Configure GitHub Copilot for Visual Studio Code to use your custom MCP server:
 
 ## Test the MCP server
 
-1. Open GitHub Copilot in Visual Studio Code and switch to agent mode.
-1. Select the **Select tools** icon to verify your **MinimalMcpServer** is available with both tools listed.
+1. Open GitHub Copilot in Visual Studio Code and switch to chat mode.
 
-    :::image type="content" source="../media/mcp/available-tools.png" alt-text="A screenshot showing the available MCP tools.":::
+1. Select the **Select tools** icon to verify your **SampleMcpServer** is available with the sample tool listed.
 
-1. Enter a prompt to run the **ReverseEcho** tool:
+    :::image type="content" source="../media/mcp/available-tools-nuget.png" alt-text="A screenshot showing the available MCP tools.":::
+
+1. Enter a prompt to run the **get_random_number** tool:
 
     ```console
-    Reverse the following: "Hello, minimal MCP server!"
+    Give me a random number between 1 and 100.
     ```
 
-1. GitHub Copilot requests permission to run the **ReverseEcho** tool for your prompt. Select **Continue** or use the arrow to select a more specific behavior:
+1. GitHub Copilot requests permission to run the **get_random_number** tool for your prompt. Select **Continue** or use the arrow to select a more specific behavior:
 
     - **Current session** always runs the operation in the current GitHub Copilot Agent Mode session.
     - **Current workspace** always runs the command for the current Visual Studio Code workspace.
     - **Always allow** sets the operation to always run for any GitHub Copilot Agent Mode session or any Visual Studio Code workspace.
 
-1. Verify that the server responds with the echoed message:
+1. Verify that the server responds with a random number:
 
     ```output
-    !revres PCM laminim ,olleH
+    Your random number is 42.
+    ```
+
+## Adding inputs and configuration options
+
+In this example, enhance the MCP server to use a configuration value set in an environment variable. This could be configuration needed for the functioning of your MCP server such as an API key, an endpoint to connect to, or a local directory path depending on your needs.
+
+1. Update the tool code to use an environment variable. Add another tool method after the `GetRandomNumber` method in `Tools/RandomNumberTools.cs`:
+
+   ```csharp
```

Comment:
Can you put the code in a separate file and reference it like this? https://github.com/dotnet/docs/blob/75198f1e19a9ac3216cef2d4a65a2cb59e39e5e6/docs/ai/quickstarts/build-mcp-server.md?plain=1#L49